SpaceTuna8/data-alpha-Guilf
# Alpha v1.1

To test script: 
add/confirm proper Authorization.JSON file is in dataComp folder
delete the graphs stored in dataComp folder
delete grouping.R file in dataComp folder
Set Config file variables to appropriate values 
Set up well names and well groupings on [google sheet](https://docs.google.com/spreadsheets/d/1fJhE1hOMqVvf5T8YHxRATOQ8QHKfujZRym2wk-tYq4I/edit#gid=0)
Run Main python script from Python IDLE
It should regenerate grouping.R and graph .png files according to the google spreadsheet
